mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -251,6 +252,9 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 							Optional:     true,
 							ValidateFunc: validation.IntBetween(0, 3600),
 							Description:  `Minimum number of seconds to wait for after a newly created instance becomes available. This value must be from range [0, 3600].`,
+						<% if version == "ga" -%>
+							Deprecated:   `This is beta only attribute`,
```

Comment:
Can we change these messages to something like:

"This attribute is currently in beta and will be removed from the google provider. Please use the google-beta provider to continue using this attribute."